### DISCUSSION TITLE
How do I create an on-disk folder structure with Markdown files like v1.x

### DISCUSSION THREAD

**Discussion by kaipee** [requarks/wiki#3361] [Help / Questions]
**Question**
In v1.x I had a folder structure containing Markdown files. I would like to know how/where I can do this.

**Host Info (please complete the following information):**
OS: Ubuntu 18.04
Wiki.js version: 2.1.113
Wikijs install : /opt/wiki
Database engine: postgres 9.5.19

**Comment by despens** [comment]
In the admin area under "Storage", you can select the target "Local File System" and set up the desired path.

**Comment by kaipee** [comment]
> In the admin area under "Storage", you can select the target "Local File System" and set up the desired path.

I've configured "Administration > Storage > Local File System" with the path /opt/wiki/pages but I'm seeing an error:

> Mar 17 13:47:20 wikijs node[1367]: 2020-03-17T13:47:20.163Z [MASTER] #033[33mwarn#033[39m: expecting a function but got [object Undefined]
> Mar 17 13:47:53 wikijs node[1367]: 2020-03-17T13:47:53.596Z [MASTER] #033[33mwarn#033[39m: Invalid Handler for Storage Target
> Mar 17 13:48:11 wikijs node[1367]: 2020-03-17T13:48:11.491Z [MASTER] #033[33mwarn#033[39m: expecting a function but got [object Undefined]
> Mar 17 13:48:20 wikijs node[1367]: 2020-03-17T13:48:20.926Z [MASTER] #033[33mwarn#033[39m: Invalid Handler for Storage Target

The filesystem is ext4

> drwxrwxr-x 6 wikijs wikijs 4096 Mar  4 20:33 /opt/wiki/pages/
